actions/visual-recognition-v3/create-classifier.js

```diff
@@ -47,21 +48,6 @@ function main(params) {
         return;
       }
     });
-    const positiveExampleClasses = Object.keys(_params).filter(key =>
```

Comment:
This is a handwritten code that should be here because the `create-classifier` takes zip files